content/interoperability.tex

```diff
@@ -111,7 +111,7 @@ \subsection{Thread Safety}
 \subsection{Mapping Process Identification Numbers}
 \label{subsec:interoperability:id}
 
-Similar to the PE identifier in \openshmem, MPI defines rank as the
+Similar to the PE number in \openshmem, MPI defines rank as the
 identification number of a process in a communicator. Both the \openshmem PE
```

Comment:
Use \ac{PE}.

Reply:
FWIW, I think we can fix all the missing `\ac{...}` macros afterward as editorial changes. (MPI itself should be an acronym, no?)

Reply:
MPI stands for `Message Passing Interface`. Should we define the acronym macro for it?

Reply:
I just found that \ac{MPI} is already defined. Replaced `MPI` with `\ac{MPI}` in the proposal.